conda/recipes/cucim/meta.yaml

```
@@ -63,7 +63,7 @@ requirements:
     {% endif %}
     - cupy >=12.0.0
     - libcucim ={{ version }}
-    - numpy 1.21
+    - numpy 1.23
```

Just for illustrative purposes, this is what it would look like to pin NumPy to 1.21 for Python pre-3.11 and to NumPy 1.23 starting with Python 3.11

```suggestion
    - numpy 1.21  # [py < 311]
    - numpy 1.23  # [py >= 311]
```

---

Is `311` a valid value? Perhaps the [documentation](https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html#environment-variables-that-affect-the-build-process) is outdated, but it seems to indicate that `37` is the last valid value for this variable.

---

Yep, here is [an example]( https://github.com/conda-forge/conda-feedstock/blob/010d41345593b8c0d98edc8fec4e8a6811c1839c/recipe/meta.yaml#L99-L100 )

That said, am ok with just using NumPy 1.23 everywhere